docs/user/dashboard/tsvb.asciidoc

```diff
@@ -53,36 +31,66 @@ When you use only {data-sources}, you are able to:
 
 * Improve performance
 
-IMPORTANT: Creating *TSVB* visualizations with an {es} index string is deprecated. To use an {es} index string, contact your administrator, or go to <<advanced-options, Advanced Settings>> and set `metrics:allowStringIndices` to `true`.  Creating *TSVB* visualizations with an {es} index string will be removed in a future release.
-Creating visualizations with only {data-sources} is the default one for new visualizations but it can also be switched for the old implementations:
+IMPORTANT: Creating *TSVB* visualizations with an {es} index string is deprecated. To use an {es} index string, contact your administrator, or go to <<advanced-options, Advanced Settings>> and set `metrics:allowStringIndices` to `true`. Creating *TSVB* visualizations with an {es} index string will be removed in a future release.
+Creating visualizations with only {data-sources} is the default.
+
+. On the dashboard, click *All types*, then select *TSVB*.
+
+. In *TSVB*, click *Panel options*, then specify the *Data* settings.
 
-. Click *Panel options*, then open the *Index pattern selection mode* options next to the *Index pattern* dropdown.
+. Open the *Index pattern selection mode* options next to the *Index pattern* dropdown.
 
-. Select *Use only Kibana index patterns*.
+. Select *Use only Kibana {data-sources}*.
 
-. From the *Index pattern* drodpown, select the index pattern, then select the *Time field*.
+. From the *Index pattern* drodpown, select the {data-source}, then select the *Time field* and *Interval*.
```

Comment:
Index pattern dropdown?

Reply:
In our slides, we decided to go with **Index pattern** dropdown.